pkg/sanitization/html.go

```diff
@@ -0,0 +1,51 @@
+package sanitization
+
+import (
+	"sync"
+
+	"github.com/microcosm-cc/bluemonday"
+)
+
+var once sync.Once
+
+var htmlSanitizerPolicy *bluemonday.Policy
+
+// SanitizeHTML takes a string representation of HTML and sanitizes it
+func SanitizeHTML(input string) string {
+	policy := getHTMLSanitizerPolicy()
+
+	output := policy.Sanitize(input)
+
+	return output
+}
+
+// getHTMLSanitizerPolicy returns the sanitization policy for HTML
+func getHTMLSanitizerPolicy() *bluemonday.Policy {
+
+	// once ensures that a policy is instantiated once. Otherwise, it is just retrieved.
+	once.Do(func() {
+		policy := createHTMLPolicy()
+		htmlSanitizerPolicy = policy
+	})
+
+	return htmlSanitizerPolicy
+}
+
+// createHTMLPolicy instantiates the standard HTML sanitization policy for the EASI application
+func createHTMLPolicy() *bluemonday.Policy {
+
+	policy := bluemonday.NewPolicy()
+	// NOTE make sure to update the allowed policy on the frontend when it is updated here as well
```

Comment:
As well as the note here, would it be worthwhile to put a similar comment in `src/components/RichTextEditor/index.tsx` near the declaration of `ALLOWED_TAGS`, as an additional reminder to keep these in sync?

Reply:
Thanks Dylan, I agree. I added it there. The code changes weren't merged when I originally added them here, but they are now